schema/tables/cryptsetup_status.yml

```diff
@@ -0,0 +1,14 @@
+name: cryptsetup_status
+description: TODO
+notes: This table is not a core osquery table. It is included as part of fleetd, the osquery manager from Fleet. Code based on work by [Kolide](https://github.com/kolide/launcher).
+platforms:
+  - linux
+columns:
+  - name: passphrase
+    # description: TODO
```

Comment:
@noahtalerman The commented-out column descriptions will cause an error when running the website tests. Column descriptions are required for YAML schema tables.

Reply:
Thanks @eashaw. All columns now have descriptions.

Heads up @rachaelshaw this PR is ready for re-review.